### ISSUE TITLE
[1.12] New world - Only males spawning

### ISSUE THREAD

**Issue by Tetrajak** [capnkirok/animania#362]
So, as the title suggests, I started a new world with the latest version (1.3.3) of Animania, and all the animals I came across were male.

**Comment by capnkirok** [comment]
That is strange since males can only spawn if a female spawns first. Still happening? Can you try a new world?

**Comment by Tetrajak** [comment]
So a few more details in an attempt to reproduce this bug;
Minecraft Version: 1.12.1
Forge Version: 14.22.1.2485

I've explored a lot further, and created several worlds, and I did eventually find female animals. I'm not sure why, but it appears that the spawn chunks, or perhaps close proximity to spawn, favours bulls over cows (it doesn't seem to be affecting other animal types). Perhaps it's just a fluke, perhaps it's just my luck, I'm not sure.
